src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let totalValue = days * 40;
+
+  if (days >= 3 && days < 7) {
+    totalValue = days * 40 - 20;
+  } else if (days >= 7) {
+    totalValue = days * 40 - 50;
```

Comment:
Similar to the previous comment, avoid redundant calculations by using the already calculated 'totalValue'.